crate/utils/src/lib.rs

```diff
@@ -29,16 +29,19 @@ pub trait DecryptionSystem {
 pub struct KeyPair(pub (Object, Object));
 impl KeyPair {
     /// Create a new `KeyPair` from a private and public key
+    #[must_use]
     pub fn new(private_key: Object, public_key: Object) -> Self {
         Self((private_key, public_key))
     }
 
     /// Get the private key
+    #[must_use]
```

Comment:
I may not be on the same page with you about the use of `must_use`. To me, inexpensive functions don't have to be tagged.

Reply:
I understand from clippy that is recommended to use it:
https://rust-lang.github.io/rust-clippy/master/index.html#/must_use_candidate:
```
The lint only checks the arguments for mutable types without looking if they are actually changed. On the other hand, it also ignores a broad range of potentially interesting side effects, because we cannot decide whether the programmer intends the function to be called for the side effect or the result. Expect many false positives. At least we don’t lint if the result type is unit or already #[must_use].
```